README.md

```diff
@@ -1,39 +1,219 @@
-# graphframes
+<img src="docs/img/GraphFrames-Logo-Large.png" alt="GraphFrames Logo" width="400"/>
+
 [![Scala CI](https://github.com/graphframes/graphframes/actions/workflows/scala-ci.yml/badge.svg)](https://github.com/graphframes/graphframes/actions/workflows/scala-ci.yml)
 [![Python CI](https://github.com/graphframes/graphframes/actions/workflows/python-ci.yml/badge.svg)](https://github.com/graphframes/graphframes/actions/workflows/python-ci.yml)
 [![pages-build-deployment](https://github.com/graphframes/graphframes/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/graphframes/graphframes/actions/workflows/pages/pages-build-deployment)
 
-
 # GraphFrames: DataFrame-based Graphs
 
-This is a package for DataFrame-based graphs on top of Apache Spark.
-Users can write highly expressive queries by leveraging the DataFrame API, combined with a new
-API for motif finding.  The user also benefits from DataFrame performance optimizations
-within the Spark SQL engine.
+This is a package for DataFrame-based graphs on top of Apache Spark. Users can write highly expressive queries by leveraging the DataFrame API, combined with a new API for network motif finding. The user also benefits from DataFrame performance optimizations within the Spark SQL engine. GraphFrames works in Java, Scala, and Python.
+
```

Comment:
Let's add a note about deprecation of GraphX in Spark 4.0? Something like "With deprecation of GraphX in Apache Spark Core GraphFrames is the recommended approach for the distributed graphs processing."?